rb/lib/selenium/webdriver/bidi/network.rb

```diff
@@ -91,7 +91,7 @@ def continue_with_request(**args)
         def continue_with_response(**args)
           @bidi.send_cmd(
             'network.continueResponse',
-            request: args[:request_id],
+            request: args[:response_id],
```

Comment:
I am not from ruby world, but seems `request_id` was correct. https://w3c.github.io/webdriver-bidi/#command-network-continueResponse

```
network.ContinueResponseParameters = {
  request: network.Request,
  ?cookies: [*network.SetCookieHeader]
  ?credentials: network.AuthCredentials,
  ?headers: [*network.Header],
  ?reasonPhrase: text,
  ?statusCode: js-uint,
}
```

Reply:
Thank you @nvborisenko I changed it because I thought it was confusing to send a request_id when working with a response, but I will roll it back since it can be confusing also when reading the spec